rosbag2_bag_v2_plugins/src/rosbag2_bag_v2_plugins/storage/rosbag_v2_storage.hpp

```diff
@@ -43,6 +43,14 @@ class RosbagV2Storage : public rosbag2_storage::storage_interfaces::ReadOnlyInte
 
   rosbag2_storage::BagMetadata get_metadata() override;
 
+  uint64_t get_bagfile_size() const override;
```

Comment:
```suggestion
  uint64_t get_bagfile_size() const final;
```

prefer `final` to `override` whenever possible.

Reply:
this comment is ok to ignore see my PR comment